src/components/Favourites.js

```diff
@@ -1,25 +1,25 @@
-import React from 'react';
-import './Favourites.css';
+import React from "react";
+import "./Favourites.css";
 
 class Favourites extends React.Component {
-	render() {
-		return (
-			<div className="Favourites">
-				<div className="Favourites-breeds">
-					<h2>Favourite Breeds</h2>
-					<p>???</p>
-					<p>???</p>
-				</div>
-				<div className="Favourites-saved">
-					<h2>Saved Photos</h2>
-					<div className="Favourites-photos">
-						<img src="http://via.placeholder.com/80x80" />
-						<img src="http://via.placeholder.com/80x80" />
-					</div>
-				</div>
-			</div>
-		)
-	}
+  render() {
+    return (
+      <div className="Favourites">
+        <div className="Favourites-breeds">
+          <h2>Favourite Breeds</h2>
+          <p>???</p>
+          <p>???</p>
+        </div>
+        <div className="Favourites-saved">
+          <h2>Saved Photos</h2>
+          <div className="Favourites-photos">
+            <img src="http://via.placeholder.com/80x80" />
+            <img src="http://via.placeholder.com/80x80" />
+          </div>
+        </div>
+      </div>
+    );
+  }
 }
 
```

Comment:
None of the changes in this file should be in this commit. Remember to check `git status` before commiting so you don't commit files you haven't changed.